commands/operator-sdk/cmd/test/cluster.go

```diff
@@ -89,7 +92,7 @@ func testClusterFunc(cmd *cobra.Command, args []string) error {
 				Name:            "operator-test",
 				Image:           args[0],
 				ImagePullPolicy: pullPolicy,
-				Command:         []string{"/go-test.sh"},
+				Command:         []string{scaffold.GoTestScriptFile},
```

Comment:
Seems like the tests are failing as it's expecting this file `go-test.sh`.

Reply:
Yep that's the issue. Good catch all.